Billing Center Cost Anomaly - Fix for Incorrect Variable Type and Missing Detail Template

srpomeroy opened the pull request:
### Description

The report variable was defined as an object, so when no report data is available, the policy evaluation fails as it is expecting an array.

Also added back in the detail template as there was too much information in the summary template.

### Issues Resolved

n/a

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

chandravurubindi reviewed (dismissed):
LGTM, there are few lines indentation needs to be corrected, so could you please look at them before merge the PR, thanks!

srpomeroy commented:
> LGTM, there are few lines indentation needs to be corrected, so could you please look at them before merge the PR, thanks!

Thanks, @chandravurubindi., Fixed the indents, added section headers, and reorganized data sources and scripts.

gonzalez reviewed (approved):
LGTM
